test/tools/ossfuzz/protoToYul.h

```diff
@@ -93,6 +101,13 @@ class ProtoConverter
 	void visit(Program const&);
 	void registerFunction(FunctionDefinition const&);
 
+	/// Creates a new scope, and optionally adds @a _funcParams to it
+	void openScope(std::vector<std::string> const& _funcParams);
```

Comment:
What exactly is optional here? Does an empty `_funcParams` have this semantic when it's being passed to `addToScope`?

Reply:
Yeah, an empty `_funcParams` has that semantic. Should I reword the comment?

Reply:
I think that would be good. Perhaps this behaviour should be also mentioned in the comment on `addToScope`.